.coveragerc

```diff
@@ -549,6 +549,7 @@ omit =
     homeassistant/components/socialblade/sensor.py
     homeassistant/components/solaredge/sensor.py
     homeassistant/components/solax/sensor.py
+    homeassistant/components/somfy/*
```

Comment:
The config flow module requires 100% test coverage. Look at hue config flow tests for examples.

Reply:
Tests has been done for the Somfy config flow. Sorry for that.